gcloud-java-dns/src/test/java/com/google/gcloud/dns/DnsBatchTest.java

```diff
@@ -146,7 +147,8 @@ public void testListZones() {
     } catch (IllegalStateException ex) {
       // expected
     }
-    batchResult.error(new DnsException(new IOException("expected")));
+    RpcBatch.Callback<ManagedZonesListResponse> capturedCallback = callback.getValue();
+    capturedCallback.onFailure(new GoogleJsonError());
```

Comment:
Nit (feel free to ignore this comment): `new GoogleJsonError()` could be made a `static final` variable.